### ISSUE TITLE
Please do not tag releace candidate as latest

### ISSUE THREAD

**Issue by codejamninja** [cheeriojs/cheerio#1261]
`1.0.0-rc.2` has bugs and should not be tagged as `latest`. Please tag it as `next`, and tag `0.22.0` as `latest`.

This issue is in response to #1260

**Comment by pgarciacamou** [comment]
I agree. Only `stable` releases should be tagged as `latest`. This tiny detail was breaking our new project when installing `latest` from `npm`.

**Comment by matthewmueller** [comment]
Hey folks,

Apologies for the issues during this transition period. Parse5 will solve a lot of outstanding bugs, but it is a breaking change for sure.

We're going to continue to support htmlparser2 because there are lots of cases where the HTML "tree correction" is not what you want or expect.  If you prefer to use `htmlparser2` parse you can do:

```js
const dom = htmlparser2.parseDOM(file.contents, options)
const $ = cheerio.load(dom)
```

**Comment by astrotim** [comment]
Can anyone help with how to configure Enzyme to accept the `htmlparser2: true` option?

**Comment by 0xdevalias** [comment]
@astrotim I opened an issue with enzyme about this: https://github.com/airbnb/enzyme/issues/2299

**Comment by meyer** [comment]
This issue persists. RC3 is tagged as latest.
